hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientWriteHandler.java

```diff
@@ -88,8 +89,8 @@ private void writeBuffer() throws IOException {
         }
     }
 
-    public void enqueueSocketWritable(SocketWritable socketWritable) {
-        writeQueue.offer(socketWritable);
+    public void enqueue(Frame frame) {
```

Comment:
Shouldn't it be `public void enqueue(Packet packet)`? We do an explicit cast anyway and the method is not a part of an implemented interface. WDYT?